src/main/java/com/github/romankh3/image/comparison/ImageComparison.java

```diff
@@ -185,6 +190,39 @@ public ImageComparison(BufferedImage expected, BufferedImage actual) {
     }
 
     /**
+     * [work for issue #153]
+     * Draw rectangles which cover the regions of the difference pixels.
+     *
+     * @return the result of the drawing.
+     */
+    public ImageComparisonResult compareImages_BMP() {
+
+        // check that the images have the same size
+        if (isImageSizesNotEqual(expected, actual)) {
```

Comment:
This logic have code duplicate. Please. provide solution for using it from one place.

Reply:
I think adding a param to compareImages() method to indicate whether using the BMP version could help... 
I mean I just tried to avoid "breaking change" at first.
Thanks for comments, I will check and fix the code for the near term.